_includes/footer.html

```diff
@@ -1,25 +1,14 @@
-
-<div class="py-5 border-top">
+<footer class="flex-shrink-0 bg-dark py-3">
   <div class="container">
-
-    <div class="row">
-
-      <div class="col-sm">
-        <ul class="list-inline">
-          <li class="list-inline-item">
-            <a href="https://www.hackerone.com/start-hacking">🔨 Powered by HackerOne</a>
-          </li>
-          <li class="list-inline-item">|</li>
-          <li class="list-inline-item">
-            <a href="{{ site.github.repository_url }}/edit/master/{{ page.path }}" title="Help improve Hacker101">💻 Edit this page</a>
```

Comment:
Should we keep this in the new footer? I always found it useful, and I know other folks have used it as well.

Reply:
Yeah, I can keep it.